builder.Makefile

```diff
@@ -57,11 +59,12 @@ protos:
 cli:
 	GOOS=${GOOS} GOARCH=${GOARCH} $(GO_BUILD) $(TAGS) -o $(BINARY_WITH_EXTENSION) ./cli
 
+cross-%:
+	GOOS=$(word 1,$(subst -, ,$(*))) GOARCH=$(word 2,$(subst -, ,$(*))) \
```

Comment:
I think for 5 values in a makefile, I'd prefer listing the 5 lines explicitely and avoid all the string manipulation that is hard to test, validate and is a nightmare to read :). 
It would also allow for target auto-completion and make it obvious what targets are supported, I'm happy to live with small duplication for this

Reply:
Yeah, I agree that it's hairy to read. Your suggestion would mean that to add a platform, we'd need to:
1. Add `cross-new-platform`
2. Add a call from `cross` to `cross-new-platform`
3. Add the new platform to `package`

If we'd like like to make it possible to build the platform from the top-level Makefile, we'd also need to add a new target there.

I guess it comes down to how often we're going to touch this code. We should likely add builds for `linux/arm` and `darwin/arm64` in the near future.